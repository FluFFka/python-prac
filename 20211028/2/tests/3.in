www=.11
print(list(slide([KeyboardInterrupt, ..., int, .3, '123', f"{www+2}"], 2)))
