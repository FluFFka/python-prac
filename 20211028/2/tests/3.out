[<class 'KeyboardInterrupt'>, Ellipsis, Ellipsis, <class 'int'>, <class 'int'>, 0.3, 0.3, '123', '123', '2.11', '2.11']
